packages/create-vite/index.js

```diff
@@ -237,6 +263,33 @@ async function init() {
   // determine template
   template = variant || framework || template
 
+  const pkgInfo = pkgFromUserAgent(process.env.npm_config_user_agent)
+  const pkgManager = pkgInfo ? pkgInfo.name : 'npm'
+
+  if (template.startsWith('custom-')) {
+    const getCustomCommand = (name) => {
+      for (const f of FRAMEWORKS) {
+        for (const v of f.variants || []) {
+          if (v.name === name) {
+            return v.customCommand
+          }
+        }
+      }
+    }
+    const customCommand = getCustomCommand(template)
+    const fullCustomCommand = customCommand
+      .replace('TARGET_DIR', targetDir)
+      .replace(/^npm /, `${pkgManager} `)
+      // Only yarn doesn't support `@version` in the `create` command
+      .replace('@latest', () => (pkgManager === 'yarn' ? '' : '@latest'))
```

Comment:
nuxt is using `npx`. Maybe we need to replace `npx` too? Or use `npm exec` (requires npm@7+) instead of `npx`?